docs/changelog.md

```diff
@@ -6,19 +6,20 @@ permalink: /changelog/
 
 # 75.0.0-SNAPSHOT (In Development)
 
-* [Commits](https://github.com/mozilla-mobile/android-components/compare/v74.0.0...master)
-* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/136?closed=1)
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v75.0.0...master)
+* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/137?closed=1)
```

Comment:
136 is correct here.

Reply:
The first line is correct but I think we should have this change as a separate PR.  The original change we're uplifting here doesn't have these two lines.